<ui version="4.0" >
 <class>MvdMainInfoPage</class>
 <widget class="QWidget" name="MvdMainInfoPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Main info</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="mainBox" >
     <property name="title" >
      <string>Main info</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="4" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="2" column="0" >
         <widget class="QLabel" name="labVersion" >
          <property name="text" >
           <string>Edition:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="MvdClearEdit" name="title" />
        </item>
        <item row="2" column="1" >
         <widget class="MvdClearEdit" name="version" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="labStorageID" >
          <property name="text" >
           <string>Storage ID:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="labOriginalTitle" >
          <property name="text" >
           <string>Original title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="labTitle" >
          <property name="text" >
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="MvdClearEdit" name="originalTitle" />
        </item>
        <item row="3" column="1" >
         <widget class="MvdClearEdit" name="storageID" />
        </item>
       </layout>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item rowspan="2" row="1" column="3" >
       <layout class="QGridLayout" >
        <item row="0" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>31</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" >
         <widget class="MvdPosterLabel" name="poster" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2" >
         <widget class="QLabel" name="posterStatus" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="contextMenuPolicy" >
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;a href="movida://select/poster">&lt;span style=" text-decoration: underline; color:#0000ff;">Set&lt;/span>&lt;/a> a movie poster&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="textFormat" >
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="labProductionYear" >
          <property name="text" >
           <string>Production year:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="productionYear" >
          <property name="minimum" >
           <number>0</number>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
          <property name="value" >
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2" >
         <widget class="QLabel" name="clearProductionYear" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="contextMenuPolicy" >
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="text" >
           <string>(&lt;a href="movida://clear/productionyear">clear&lt;/a> production year)</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="labReleaseYear" >
          <property name="text" >
           <string>Release year:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="releaseYear" >
          <property name="minimum" >
           <number>0</number>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
          <property name="value" >
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="clearReleaseYear" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="contextMenuPolicy" >
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="text" >
           <string>(&lt;a href="movida://clear/releaseyear">clear&lt;/a> release year)</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>121</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>47</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="labLength" >
          <property name="text" >
           <string>Running time:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QSpinBox" name="lengthMinutes" >
          <property name="maximum" >
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2" >
         <widget class="QLabel" name="clearRunningTime" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="contextMenuPolicy" >
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">minutes (&lt;a href="movida://clear/runningtime">&lt;span style=" text-decoration: underline; color:#0000ff;">clear&lt;/span>&lt;/a> running time)&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="3" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="ratingBox" >
     <property name="title" >
      <string>My rating</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="ratingStatus" >
        <property name="focusPolicy" >
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text" >
         <string>Rating</string>
        </property>
        <property name="textInteractionFlags" >
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="MvdRatingWidget" name="ratingLabel" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::TabFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>461</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>508</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MvdClearEdit</class>
   <extends>QLineEdit</extends>
   <header>mvdshared/clearedit.h</header>
  </customwidget>
  <customwidget>
   <class>MvdRatingWidget</class>
   <extends>QLabel</extends>
   <header>ratingwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MvdPosterLabel</class>
   <extends>QLabel</extends>
   <header>posterlabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>originalTitle</tabstop>
  <tabstop>version</tabstop>
  <tabstop>storageID</tabstop>
  <tabstop>productionYear</tabstop>
  <tabstop>clearProductionYear</tabstop>
  <tabstop>releaseYear</tabstop>
  <tabstop>clearReleaseYear</tabstop>
  <tabstop>lengthMinutes</tabstop>
  <tabstop>clearRunningTime</tabstop>
  <tabstop>poster</tabstop>
  <tabstop>posterStatus</tabstop>
  <tabstop>ratingLabel</tabstop>
  <tabstop>ratingStatus</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
